src/common/common.css

```diff
@@ -71,6 +71,12 @@ body {
 .success::before {
   background-image: url('/common/img/check.svg');
 }
+.icon-dismiss {
```

Comment:
There are still some styles missing for hovering e.g.
AFAIK the correct styles are described as a [ghost](https://design.firefox.com/photon/components/buttons.html#ghost-3) button, but [this site](https://design.firefox.com/photon/components/message-bars.html#type-specific-style9) also shows some screenshots (when not hovered).

Reply:
(I don't really know whether it should have a transparent background or really that grey one as specified)

Reply:
So [I've asked them about this](https://github.com/FirefoxUX/photon/issues/298).